policyengine_us/parameters/gov/states/az/tax/income/exemptions/amount/age.yaml

```diff
@@ -0,0 +1,15 @@
+description: Arizona provides an exemption of this amount per aged filer or spouse.
+metadata:
+  unit: currency-USD
+  period: year
+  label: Arizona aged exemption amount
+  reference: 
+    - title: Arizona Form 140 (2022) Page2 (38)
+      href: https://azdor.gov/forms/individual/form-140-resident-personal-income-tax-form-calculating
+    - title: Arizona Form 140 (2021) Page2 (38)
+      href: https://azdor.gov/forms/individual/form-140-resident-personal-income-tax-form-calculating
+    - title: 43-1023. Arizona Exemptions for blind persons and persons sixty-five years of age or older
```

Comment:
```suggestion
    - title: 43-1023. Arizona Exemptions for blind persons and persons sixty-five years of age or older. (E)
```

Reply:
point to the exact part of the legal code where the number can be found -> all yaml files

Reply:
done